zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/service/JWTPermissionsServiceImpl.java

```diff
@@ -0,0 +1,282 @@
+package com.github.onsdigital.zebedee.permissions.service;
+
+import com.github.onsdigital.exceptions.JWTVerificationException;
+import com.github.onsdigital.zebedee.json.CollectionDescription;
+import com.github.onsdigital.zebedee.json.PermissionDefinition;
+import com.github.onsdigital.zebedee.model.Collection;
+import com.github.onsdigital.zebedee.session.model.Session;
+import com.github.onsdigital.zebedee.session.service.Sessions;
+import com.github.onsdigital.zebedee.teams.model.Team;
+import com.github.onsdigital.zebedee.user.model.User;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Set;
+
+import static com.github.onsdigital.zebedee.logging.CMSLogEvent.warn;
+
+public class JWTPermissionsServiceImpl implements PermissionsService {
```

Comment:
Javadoc on class. Doesn't need to be hugely detained just outline that this implementation uses the JWT sessions implementation.